### ISSUE TITLE
Incorrect sample to start SSH service using ibmi_tcp_server_service module?

### ISSUE THREAD

**Issue by syahrul-aiman** [IBM/ansible-for-i#164]
Hi team

Looking at the following module:
https://github.com/IBM/ansible-for-i/blob/devel/plugins/modules/ibmi_tcp_server_service.py

This is the example task given:
```yaml
- name: start tcp server service
  ibmi_ibm.power_ibmi.ibmi_tcp_server_service:
    name_list: ['*SSH', '*HTTP']
    state: 'started'
    joblog: True
```

I'm a bit confused on this sample given as the connection to the IBMi node is via SSH. So how is this module being executed if the SSH service has not been started?

Am I missing something?

Thanks!

**Comment by robgjertsen1** [comment]
This example for ibmi_tcp_server_service is incorrect to include ssh in the list for starting a service. That needs to be performed outside of an ansible command. Probably fine to stop the ssh service this way.
Documentation should remove this from the name_list and could add something else like *DIRSRV .

**Comment by syahrul-aiman** [comment]
Hi @robgjertsen1,

Thank you so much for clarifying this. This makes more sense.

Thanks again and have a nice day!

**Comment by krka01** [comment]
@syahrul-aiman @robgjertsen1 The sample is correct, it is possible to stop and start the SSHD daemon from the same ssh connection as stopping the SSHD will not end active ssh connections.

We use the ibmi_ibm.power_ibmi.ibmi_tcp_server_service to both stop and then start the SSHD in the same playbook after we have installed PTFs for the SSH server or modify the sshd_config file and it works perfectly fine.

I wish all TCP servers worked this way.

**Comment by robgjertsen1** [comment]
Thanks for pointing out your use case with using the same playbook for stopping and starting ssh for update of sshd. I haven't  tried this yet, but makes sense that active ssh connection is maintained. It would probably be helpful to point this out in the documentation versus the isolated code snippet for starting ssh.

**Comment by robgjertsen1** [comment]
Fixed with release 2.0.1.
